src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setColor(getRandomColor());
+        ball.setNumber(new Random().nextInt(100));

```

Comment:
**All magic numbers in your code should be constants.**

Make Random a class-level variable. (As in ColorSupplier )